deploy/eck-stack/examples/metricbeat_hosts.yaml

```diff
@@ -0,0 +1,219 @@
+eck-elasticsearch:
+  enabled: true
+
+  # Name of the Elasticsearch resource.
+  #
+  fullnameOverride: quickstart
+
+  # Version of Elasticsearch.
+  #
+  version: 8.2.3
+
+  nodeSets:
+  - name: default
+    count: 3
+    config:
+      # Comment out when setting the vm.max_map_count via initContainer, as these are mutually exclusive.
+      # For production workloads, it is strongly recommended to increase the kernel setting vm.max_map_count to 262144
+      # and leave node.store.allow_mmap unset.
+      # ref: https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-virtual-memory.html
+      #
+      node.store.allow_mmap: false
+    volumeClaimTemplates:
+    - metadata:
+        name: elasticsearch-data
+      spec:
+        accessModes:
+        - ReadWriteOnce
+        resources:
+          requests:
+            storage: 100Gi
+        # Adjust to your storage class name
+        #
+        # storageClassName: local-storage
+
+eck-kibana:
+  enabled: true
+
+  # Name of the Kibana resource.
+  #
+  fullnameOverride: quickstart
+  
+  # Version of Kibana.
+  #
+  version: 8.2.3
+  
+  spec:
+    # Count of Kibana replicas to create.
+    #
+    count: 1
+  
+    # Reference to ECK-managed Elasticsearch resource, ideally from {{ "elasticsearch.fullname" }}
+    #
+    elasticsearchRef:
+      name: quickstart
+
+eck-beats:
+  enabled: true
+  name: metricbeat
+  spec:
+    type: metricbeat
```

Comment:
I don't think it is possible, but still asking 😄 : what if, as part of a same `eck-stack` Helm release, I want to deploy other Beats? Is it possible to manage, let say, `Filebeat` and `Metricbeat` as part of the same release? My understanding is that this kind of behaviour is possible with https://github.com/elastic/cloud-on-k8s/pull/5889

Reply:
The only way to do this, with the current setup of this Chart (not allowing a list of beats), would be to clone this repo, and update the `eck-stack/Chart.yaml` and add additional `dependencies` on the eck-beats chart, named with `aliases`, and install locally

(note the '.' meaning use local files)
```
helm install eck-stack .  --values /path/to/user/values.yaml
```